Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.18-alpine3.16 as builder
+
+RUN apk add ca-certificates --no-cache make git && update-ca-certificates
+
+WORKDIR /workspace
+
+COPY . .
+
+RUN make build
+
+FROM alpine:3.16.0
+
+COPY --from=builder /workspace/obsctl /usr/bin/obsctl
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
Maybe adding an `ENTRYPOINT` would be cool here?

Reply:
`ENTRYPOINT` doesn't work well with a stateful command like `obsctl`, @saswatamcode. For the image to be useful, the user has to run a bunch of commands to setup the API, login with a tenant, change the context, and then finally use the metrics commands, for example.